Formula/liblcf.rb

```diff
@@ -17,31 +16,32 @@ class Liblcf < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "34b0eea36554d5c51a76a48fbb100a302a0986fc38d8cee84df55864c2e57775"
   end
 
+  depends_on "cmake" => :build
   depends_on "pkg-config" => :build
   depends_on "icu4c"
 
   uses_from_macos "expat"
 
   def install
-    system "./configure", "--disable-dependency-tracking",
-                          "--disable-silent-rules",
-                          "--prefix=#{prefix}"
-    system "make"
-    system "make", "install"
+    mkdir "build" do
+      system "cmake", "..", *std_cmake_args
+      system "make"
+      system "make", "install"
+    end
```

Comment:
```suggestion
    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
    system "cmake", "--build", "build"
    system "cmake", "--install", "build"
```
Let's follow the template in `brew create --cmake`.